.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "go.inferGopath": false
+}
```

Comment:
seems like this unrelated change made it into the PR

Reply:
yes, will remove it

Reply:
Done.